java/src/org/openqa/selenium/devtools/v127/v127Network.java

```diff
@@ -127,7 +127,7 @@ public Either<HttpRequest, HttpResponse> createSeMessages(RequestPaused pausedRe
       } catch (DevToolsException e) {
         // Redirects don't seem to have bodies
         int code = pausedReq.getResponseStatusCode().orElse(HTTP_OK);
-        if (code < 300 && code > 399) {
+        if (code < 300 || code > 399) {
```

Comment:
This would complain about http 200.

Should be `if (code >= 300 && code <= 399) {` in my mind.

Reply:
I see, thanks, fixed